File: Admin_p/Admin_p.Web/Modules/Identity/AspNetUsers/AspNetUsersForm.cs
using Serenity.ComponentModel;
using System;

namespace Admin_p.Identity.Forms;

[FormScript("Identity.AspNetUsers")]
[BasedOnRow(typeof(AspNetUsersRow), CheckNames = true)]
public class AspNetUsersForm
{
    public string FirstName { get; set; }
    public DateTime CreatedAt { get; set; }
    public string Role { get; set; }
    public string ShippingAddress { get; set; }
    public string PhoneNumber { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string UserName { get; set; }
    public string NormalizedUserName { get; set; }
    public string Email { get; set; }
    public string NormalizedEmail { get; set; }
    public bool EmailConfirmed { get; set; }
    public string PasswordHash { get; set; }
    public string SecurityStamp { get; set; }
    public string ConcurrencyStamp { get; set; }
    public bool PhoneNumberConfirmed { get; set; }
    public bool TwoFactorEnabled { get; set; }
    public DateTimeOffset LockoutEnd { get; set; }
    public bool LockoutEnabled { get; set; }
    public int AccessFailedCount { get; set; }
}

File: Admin_p/Admin_p.Web/Modules/ServerTypes/Identity/AspNetUsersForm.ts
import { PrefixedContext, StringEditor, DateEditor, BooleanEditor, IntegerEditor, initFormType } from '@serenity-is/corelib';

export interface AspNetUsersForm {
    FirstName: StringEditor;
    CreatedAt: DateEditor;
    Role: StringEditor;
    ShippingAddress: StringEditor;
    PhoneNumber: StringEditor;
    UpdatedAt: DateEditor;
    UserName: StringEditor;
    NormalizedUserName: StringEditor;
    Email: StringEditor;
    NormalizedEmail: StringEditor;
    EmailConfirmed: BooleanEditor;
    PasswordHash: StringEditor;
    SecurityStamp: StringEditor;
    ConcurrencyStamp: StringEditor;
    PhoneNumberConfirmed: BooleanEditor;
    TwoFactorEnabled: BooleanEditor;
    LockoutEnd: StringEditor;
    LockoutEnabled: BooleanEditor;
    AccessFailedCount: IntegerEditor;
}

export class AspNetUsersForm extends PrefixedContext {
    static readonly formKey = 'Identity.AspNetUsers';
    private static init: boolean;
    
    constructor(prefix: string) {
        super(prefix);
        if (!AspNetUsersForm.init)  {
            AspNetUsersForm.init = true;
            
            var w0 = StringEditor;
            var w1 = DateEditor;
            var w2 = BooleanEditor;
            var w3 = IntegerEditor;

            initFormType(AspNetUsersForm, [
            'FirstName', w0,
            'CreatedAt', w1,
            'Role', w0,
            'ShippingAddress', w0,
            'PhoneNumber', w0,
            'UpdatedAt', w1,
            'UserName', w0,
            'NormalizedUserName', w0,
            'Email', w0,
            'NormalizedEmail', w0,
            'EmailConfirmed', w2,
            'PasswordHash', w0,
            'SecurityStamp', w0,
            'ConcurrencyStamp', w0,
            'PhoneNumberConfirmed', w2,
            'TwoFactorEnabled', w2,
            'LockoutEnd', w0,
            'LockoutEnabled', w2,
            'AccessFailedCount', w3,
            ]);
        }
    }
}